Makefile

```diff
@@ -44,11 +43,9 @@ codecov-test:
 	courtney/courtney -v -o coverage/coverage.out ./...
 	@cd internal/db && $(MAKE) rm-notest
 
-tidy: ## add missing and remove unused modules
-	 go mod tidy
-
 format: ## run go formatter
 	gofumpt -l -w .
+	go mod tidy
```

Comment:
I think implicitly running `go mod tidy` on the `format` command is unnecessary, and obfuscates what the actual command is doing. I personally prefer having `tidy` and `format` separated.